README.md

```diff
@@ -47,14 +47,14 @@ In this section we'll use the Azure Portal to create the required resources to h
 
 1. Ensure the _Subscription_ and _Resource Group_ are correct, then provide the following configuration for the storage account:
 
-   - _Name_ - `my-keystone-app`
+   - _Name_ - `mykeystoneapp`
    - _Region_ - Select an appropriate region
    - _Performance_ - `Standard`
    - _Redundancy_ - Select the appropriate level of redundancy for your files
 
 1. Click **Review + create** then **Create**
 
-1. Navigate back to the Resource Group and click **Create** then search for _Azure Database for Postgres_ and click **Create**
```

Comment:
Should be `Azure Database for PostgreSQL` otherwise shows a bunch of other results before the one we want:

<img width="1572" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/737821/137425320-4b7ae625-e2a2-46d8-9e5e-68bf727fdf70.png">